### PR TITLE
Improve some grammar.

### PR DESCRIPTION
**Pull request by Maviza101**
This PR aims to correct some of the grammar in the docs for the VS Code Extensions API.

### CODE REVIEW

File: api/index.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ If you'd like to have a more comprehensive overview of the Extension API, refer
...
-- **Extension Capabilities** dissects VS Code's vast API into smaller categories and points you to more detailed topics.
-- **Extension Guides** includes guides and code samples that explains specific usages of VS Code Extension API.
-- **Language Extensions** illustrates how to add support for a programming language with guides and code samples.
-- **Advanced Topics** explains advanced concepts such as [Extension Host](/api/advanced-topics/extension-host), [Supporting Remote Development](/api/advanced-topics/remote-extensions), and [Proposed API](/api/advanced-topics/using-proposed-api).
-- **References** contains exhaustive references for the [VS Code API](/api/references/vscode-api), [Contribution Points](/api/references/contribution-points), and many other topics.
+- **Getting Started**: teaches fundamental concepts for building extensions with the [Hello World](https://github.com/Microsoft/vscode-extension-samples/tree/master/helloworld-sample) sample.
```
Comment (gregvanl) [inline api/index.md]: This is the section title and is "Get Started".
We prefer to have the elements read as sentences and not have the colons.
The three grammar changes look good if you want to submit an updated PR.

Comment (Maviza101) [reply api/index.md]: Roger that. Updating the PR now.

**Comment by gregvanl** [comment]
Thanks!

**Comment by Maviza101** [comment]
Sure thing. Glad to help!

**Pull request merged**